### ISSUE TITLE
One error in the training pipeline

### ISSUE THREAD

**Issue by ziruizhu** [facebookresearch/tacto#25]
In https://github.com/facebookresearch/tacto/blob/master/experiments/grasp_stability/train.py, you should call self.model.train() before training and self.model.eval() before evaluation. There are batchnorm operations in the resnet.

**Comment by wx405557858** [comment]
Updated, thank you for pointing it out!
